docs/source/users.rst

```diff
@@ -0,0 +1,48 @@
+.. _users:
+
+================================================================================
+Software and organizations using or contributing to PROJ
+================================================================================
+
+Free and open source software
+-----------------------------
+
+- `GDAL <https://gdal.org>`_ Translator library for raster and vector geospatial data formats, used by > 100 other software
+- `GRASS GIS <http://grass.osgeo.org>`_  A raster/vector open source GIS
+- `libgeotiff <https://github.com/OSGeo/libgeotiff>`_ Library to read/write GeoTIFF keys
+- `MapServer <http://mapserver.org/index.html>`_  A popular web mapping application
+- `PDAL <https://pdal.io>`_  Point Cloud Data Abstraction Library
+- `PostGIS <http://www.postgis.net>`_ Spatial database extender for PostgreSQL
+- `pyproj <https://pyproj4.github.io/pyproj>`_ Python interface to PROJ
+- `QGIS <http://www.qgis.org>`_ A cross platform desktop GIS.
+- `R <http://www.r-project.org>`_ A free software environment for statistical computing and graphics
+
+Proprietary licensed software
+-----------------------------
+
+- `FME <http://www.safe.com>`_  A GIS translator package. Includes a PROJ based transformer.
+
+Geodesic organizations
+----------------------
+
+- `Agence for Data Supply and Infrastructure of Denmark (SDFI) <https://eng.sdfi.dk/>`_ 
+- `Land Information New Zealand (LINZ) <https://www.linz.govt.nz/>`_
+- `National Land Survey of Finland (NLS) <https://www.maanmittauslaitos.fi/en>`_
```

Comment:
You can for sure put Norway, Sweden and Iceland here too.

Reply:
done

Reply:
... and s/Agence for.../Agency for.../

Reply:
> ... and s/Agence for.../Agency for.../

fixed